selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/DeleteOrganizationByBulkTest.java

```diff
@@ -38,88 +36,83 @@
  *
  * @author Ann Shumilova
  */
-public class BulkOrganizationDeletionTest {
-  private static final Logger LOG = LoggerFactory.getLogger(BulkOrganizationDeletionTest.class);
+public class DeleteOrganizationByBulkTest {
+  private static final String ORG_NAME1 = generate("organization", 5);
+  private static final String ORG_NAME2 = generate("organization", 5);
 
   private List<OrganizationDto> organizations;
-  private OrganizationDto organization1;
-  private OrganizationDto organization2;
-
-  @Inject private OrganizationListPage organizationListPage;
-  @Inject private NavigationBar navigationBar;
-  @Inject private ConfirmDialog confirmDialog;
-  @Inject private Dashboard dashboard;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
+  @Inject private OrganizationListPage organizationListPage;
   @Inject private AdminTestUser adminTestUser;
+  @Inject private NavigationBar navigationBar;
+  @Inject private ConfirmDialog confirmDialog;
+  @Inject private Dashboard dashboard;
 
   @BeforeClass
   public void setUp() throws Exception {
-    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
-
-    String organizationName1 = NameGenerator.generate("organization", 5);
-    String organizationName2 = NameGenerator.generate("organization", 5);
-
-    organization1 = testOrganizationServiceClient.create(organizationName1);
-    organization2 = testOrganizationServiceClient.create(organizationName2);
+    testOrganizationServiceClient.create(ORG_NAME1);
+    testOrganizationServiceClient.create(ORG_NAME2);
     organizations = testOrganizationServiceClient.getAll();
+
+    dashboard.open(adminTestUser.getName(), adminTestUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(organization1.getId());
-    testOrganizationServiceClient.deleteById(organization2.getId());
+    for (OrganizationDto organization : organizations)
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.